### PR TITLE
Fix ujson python fuzzing.

### PR DESCRIPTION
**Pull request by inferno-chromium**

### CODE REVIEW

**Comment by inferno-chromium** [comment]
Cannot find any other workarounds to LD_PRELOAD requirements, patchelf does not help, if you have ideas to try, please try.

File: projects/ujson/build.sh [hunk 1]
```diff
@@ -15,10 +15,25 @@
...
 
-# Build and install the project package.
+# Build and install atheris (using current CFLAGS, CXXFLAGS).
+pip3 install --force-reinstall atheris
+
+# Build and install project (using current CFLAGS, CXXFLAGS).
 pip3 install .
 
+# Bundle ASAN runtime lib in $OUT.
+sanitizer_runtime="libclang_rt.asan-x86_64.so"
```
Comment (oliverchang) [inline projects/ujson/build.sh:25]: Should we put all of this into a common helper in base-builder? 

Also, if a project uses bazel, will it still work well with this?

**Review by oliverchang — APPROVED** [review]
LGTM, but unsure if this will work under minijail. I don't think /bin/bash is available in our minijail environment either.

**Comment by oliverchang** [comment]
I think the wrapper approach is still cleaner here... Would it be possible to rewrite it so that it works for plain /bin/sh? that is available in the minijail environment.

**Comment by inferno-chromium** [comment]
> I think the wrapper approach is still cleaner here... Would it be possible to rewrite it so that it works for plain /bin/sh? that is available in the minijail environment.

yes done. thinking more, yes sticking with old wrapper as it keeps things clean for clusterfuzz.

**Review by oliverchang — APPROVED** [review]
thanks!

**Pull request merged**